src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/resources/MacResources.properties

```diff
@@ -91,7 +91,7 @@ message.preparing-scripts=Preparing package scripts.
 message.preparing-distribution-dist=Preparing distribution.dist: {0}.
 message.signing.pkg=Warning: For signing PKG, you might need to set "Always Trust" for your certificate using "Keychain Access" tool.
 message.setfile.dmg=Setting custom icon on DMG file skipped because 'SetFile' utility was not found. Installing Xcode with Command Line Tools should resolve this issue.
-message.install-dir-ignored=Warning: "--install-dir" is not supported by DMG and will be default to /Applications.
+message.install-dir-ignored=Warning: "--install-dir" is not supported by DMG and will be default to {0}.
```

Comment:
I think it is better to change message to "Warning: "--install-dir" value "{0}" is not supported by DMG and will be default to {1}." Currently it sounds that CLI option "--install-dir" is not supported, but it will work if it points to default location.

Reply:
I like this one and yes lets change it: `Warning: "--install-dir" option is ignored for DMG packaging. The installation directory will default to {0}.`